src/Form/Question.php

```diff
@@ -169,10 +169,13 @@ private function prepareInput(&$input)
 
             $is_extra_data_valid = $question_type::validateExtraDataInput($extra_data);
 
-            if (!$is_extra_data_valid) {
+            if ($extra_data === false) {
```

Comment:
Are you sure of this change? `$is_extra_data_valid` is no more used, meaning that an invalid input may not be handled as expected.